src/components/Input/Input.tsx

```diff
@@ -12,6 +12,7 @@ export interface InputProps extends ComponentPropsWithoutRef<'input'> {
   icon?: JSX.Element;
   customClass?: string;
   onClickIcon?: () => void;
+  rightText?: string;
```

Comment:
to allow more flexibility

```suggestion
  rightText?: JSX.Element;
```

Reply:
JSX.Element cannont be a string, so it will be ReactNode type